src/VirtoCommerce.CatalogModule.Web/Scripts/blades/dynamicAssociations/dynamicAssociations-detail.js

```diff
@@ -72,17 +72,22 @@ angular.module('virtoCommerce.catalogModule')
         };
 
         $scope.outputTuning = function () {
+            const rulesBlock = _.find(blade.currentEntity.expressionTree.children, x => x.id === $scope.BlockMatchingRules);
```

Comment:
I think categories for sorting should come from Resulting rule

Reply:
changed